resources/case_studies/Insurance/Bid.obs

```diff
@@ -13,20 +13,21 @@ main contract Bid {
     }
 
     // expires after a certain amount of time
-    state Unavailable {}
+    state Expired {}
```

Comment:
My intuition is that the state declarations should go at the top of the contract (above the constructor).